### ISSUE TITLE
nano-minibuffer implicitly depends on nano-color

### ISSUE THREAD

**Issue by rprechelt** [rougier/nano-emacs#80]
The *optional* `nano-minibuffer.el` uses `material-color` on [L54](https://github.com/rougier/nano-emacs/blob/master/nano-minibuffer.el#L54) but does not explicitly load `nano-colors.el` which is another *optional* module and therefore causes a backtrace error if your init file doesn't already require `nano-colors`. 

For ease of use by new emacs users, I think it would be better if:

1. A sentence is added to the description of `nano-minibuffer.el` in the README that `nano-colors.el` is also required and must be loaded first, **or**
2. `nano-colors.el` is explicitly loaded by `nano-minibuffer.el`, **or**
3. We replace the call to `material-color` with one from `nano-basecolors.el`. 

I'll happily make an MR if you have an opinion about which of these options you'd like to see merged.

**Comment by rougier** [comment]
Good catch, thank you. I think option 3 might be the best and the most compatible with dark/light themes. If you can make a PR that would be great.
